CHANGES.md

```diff
@@ -15,6 +15,8 @@
 ### :bug: Bugs fixed
 
 - Fix pencil loader [Taiga #8348](https://tree.taiga.io/project/penpot/issue/8348)
+- Fix components are not dragged from the group to the assets tab [Taiga #8273](https://tree.taiga.io/project/penpot/issue/8273)
```

Comment:
If this PR is against develop the changes should be in 2.2.0, if those changes are for 2.1.1 then it should be merged to staging

Reply:
fixed